code/sql.c

```diff
@@ -54,7 +54,7 @@ int CSQLInterface::IQuery(const char *query)
 	
 	if(mysql_query(connection, query))
 	{
-		RS.Bug("MYSQL IQuery Error: %s on query %s", mysql_error(connection), query);
+		RS.Logger.Warn("MYSQL IQuery Error: {} on query {}", mysql_error(connection), query);
```

Comment:
This maybe could be an error too? I'm not 100% on this so whatever you think

Reply:
Fixed in latest commit.